verifier/src/report_body.rs

```diff
@@ -117,11 +117,11 @@ impl ConfigIdVerifier {
         Self { expected_id }
     }
 }
-impl Verifier<ReportBody> for ConfigIdVerifier {
+impl Verifier<ConfigId> for ConfigIdVerifier {
     type Error = VerificationError;
-    fn verify(&self, evidence: &ReportBody) -> CtOption<Self::Error> {
+    fn verify(&self, evidence: &ConfigId) -> CtOption<Self::Error> {
         let expected = self.expected_id.clone();
-        let actual = evidence.config_id();
+        let actual = evidence.clone();
```

Comment:
Not fond of this cloning multiple times, but I think the use case of the verifier is narrow enough that the extra cloning of the data isn't likely to be an issue for the entire system